R/deployApp.R

```diff
@@ -4,6 +4,10 @@
 #' [RMarkdown][rmarkdown::rmarkdown-package] document, a plumber API, or HTML
 #' content to a server.
 #'
+#' In order to ignore specific files in the working directory while deploying,
+#' the files must be listed in the .rscignore file. This file must have one
+#' file or directory per line with no support for wildcards.'
+#'
```

Comment:
I think this should be moved to a less prominent location, but I'll take care of that after merge.